Debug

yohskua opened the pull request:
- Fils de Thengel, mergez moi ça

yohskua committed:
fix: call a different function when displaying node representation with graphviz

yohskua committed:
feat: train only when "minimum_training_size" of new experience samples have been accumulated

yohskua committed:
refacto: abstract training on queue away from run function for more readability

yohskua committed:
style: do some black reformatting for long string

yohskua committed:
fix style: correct rebase error and black formatting

dalkio commented on src/run.py:
```diff
@@ -72,6 +72,28 @@ def play_game(
+    loss, updated, iteration = train_samples(
+        run_id, states_batch, [policies_batch, rewards_batch]
+    )
+    print("Training took {:.2f} seconds".format(time.time() - training_starting_time))
```
```suggestion
    print(f"Training took {time.time() - training_starting_time:.2f} seconds")
```

dalkio commented on src/run.py:
```diff
@@ -146,31 +170,24 @@ def play_game(
                 f"iteration_{iteration}",
             )
+            run_new_experience = 0
+
```
Remove blank line

yohskua committed:
style: remove blank line

yohskua committed:
Update src/run.py

Co-authored-by: Côme Arvis <come.arvis@neuronest.net>

yohskua committed:
feat: add proper behavior to trigger trainings
- trigger a first train after having accumulated minimum_training_size samples
- trigger trains that follow 1st train every time at least minimum_delta_size new samples have been accumulated
